lib/msf/core/session/provider/single_command_shell.rb

```diff
@@ -93,6 +93,14 @@ def shell_command_token(cmd, timeout=10)
     output
   end
 
+  def to_cmd(cmd, args)
+    if platform == 'windows'
```

Comment:
TODO: I"m not sure if this is the right fix, but if it's passing more tests - then it's definitely putting us more in the right direction :+1: